### PR TITLE
feat(integ-runner): add option to show deployment output

### PR DESCRIPTION
**Pull request by mrgrain**
Currently no further output is emitted from the workers which makes it difficult to diagnose any issues.
This change addresses the problem by turning the existing `verbose` flag into a count, that can indicate the requested level of debug output.

`-v` - the current level of verbosity
`-vv` - print the deployment output
`-vvv` - verbose deployment output
`-vvvv` - additional debug output from deployment

----

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md)

### Adding new Unconventional Dependencies:

* [ ] This PR adds new unconventional dependencies following the process described [here](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md/#adding-new-unconventional-dependencies)

### New Features

* [ ] Have you added the new feature to an [integration test](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md)?
	* [ ] Did you use `yarn integ` to deploy the infrastructure and generate the snapshot (i.e. `yarn integ` without `--dry-run`)?

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/integ-runner/lib/workers/common.ts [hunk 1]
```diff
@@ -152,9 +152,19 @@ export interface IntegTestOptions {
    * Whether to enable verbose logging
    *
    * @default false
+   * @deprecated - use `verbosity` instead
```
Comment (mrgrain) [inline packages/@aws-cdk/integ-runner/lib/workers/common.ts:155]: Since this is an experimental API, should I just remove the prop?

Comment (mrgrain) [reply packages/@aws-cdk/integ-runner/lib/workers/common.ts:155]: What do you think @corymhall ?

Comment (TheRealAmazonKendra) [reply packages/@aws-cdk/integ-runner/lib/workers/common.ts:155]: I would be in favor of removing it.

Comment (corymhall) [reply packages/@aws-cdk/integ-runner/lib/workers/common.ts:155]: yeah we can just remove it.

**Review by corymhall — APPROVED** [review]

**Comment by mrgrain** [comment]
@corymhall @TheRealAmazonKendra I've remove the deprecated prop. Let me know if we need anything before we can merge this PR.

**Pull request merged**